x-pack/plugins/osquery/cypress/e2e/all/custom_space.cy.ts

```diff
@@ -5,20 +5,22 @@
  * 2.0.
  */
 
-import { ROLE, login } from '../../tasks/login';
+import { tag } from '../../tags';
 import { navigateTo } from '../../tasks/navigation';
 import {
   checkActionItemsInResults,
   checkResults,
   inputQuery,
   selectAllAgents,
   submitQuery,
+  isServerless,
 } from '../../tasks/live_query';
 import { loadSpace, loadPack, cleanupPack, cleanupSpace } from '../../tasks/api_fixtures';
 
+const spaces = isServerless ? ['default'] : ['default', 'custom-space'];
 describe('ALL - Custom space', () => {
-  ['default', 'custom-space'].forEach((spaceName) => {
-    describe(`[${spaceName}]`, () => {
+  spaces.forEach((spaceName) => {
```

Comment:
nit: with this solution, we're using the `isServerless` as condition, which we want to avoid. also, on serverless there won't exist a test for `custom-space`, instead of just being skipped as with tags.

another, condition-free solution could be adding the tags to the spaces array, e.g.:
```ts
const testDescriptions = [
    { space: 'default', tags: [tag.ESS, tag.SERVERLESS]},
    { space: 'custom-spaces', tags: [tag.ESS]},
]
```

and than this can be used for the `forEach`